src/components/TitleComponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+
+function RenderTitle({title}) {
+    return (
+        <div class="intro">
```

Comment:
What is this intro class for, also in react, we use `className` instead of `class`

Reply:
I havent messed with any of the classes yet until we incorporated bootstrap.... that intro class is part of the css in joes starter code... so i left it as a placeholder until we incorporate that css doc